newsfragments/3852.minor

```diff
@@ -0,0 +1 @@
+
```

Comment:
Now that Python 3 is publicly available, bug fixes to Python 3 support are real bugfixes. So this should be `.bugfix` with appropriate summary.